docs/reference/connector/apis/check-in-connector-api.asciidoc

```diff
@@ -1,12 +1,11 @@
 [[check-in-connector-api]]
 === Check in connector API
-
-preview::[]
-
 ++++
 <titleabbrev>Check in a connector</titleabbrev>
 ++++
 
+preview::[]
```

Comment:
If we want to follow this pattern, there are couple more files to change:
- `delete-connector-api.asciidoc`
- `delete-connector-sync-job-api.asciidoc`
- `list-connectors-api.asciidoc`
- `list-connector-sync-jobs-api.asciidoc`

Reply:
Did I not get all these? 😃 :cat_furiously_typing: